### ISSUE TITLE
Error during create dulo-app

### ISSUE THREAD

**Issue by ricardovdbroek** [jaredtbrown/create-dulo-app#1]
```bash
Ricardos-MacBook-Pro-3:test ricardovandenbroek$ yarn create dulo-app
yarn create v1.22.10
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 🔨  Building fresh packages...
success Installed "create-dulo-app@0.6.1" with binaries:
      - create-dulo-app
✔ Project name: … test-app
✔ Select a framework: › react
✔ Do you want to use Typescript? … yes
node:internal/errors:465
    ErrorCaptureStackTrace(err);
    ^

TypeError [ERR_IMPORT_ASSERTION_TYPE_MISSING]: Module "file:///Users/ricardovandenbroek/.config/yarn/global/node_modules/create-dulo-app/templates/react-typescript/package.json" needs an import assertion of type "json"
    at new NodeError (node:internal/errors:372:5)
    at validateAssertions (node:internal/modules/esm/assert:82:15)
    at defaultLoad (node:internal/modules/esm/load:24:3)
    at ESMLoader.load (node:internal/modules/esm/loader:407:26)
    at ESMLoader.moduleProvider (node:internal/modules/esm/loader:326:22)
    at new ModuleJob (node:internal/modules/esm/module_job:66:26)
    at ESMLoader.#createModuleJob (node:internal/modules/esm/loader:345:17)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:304:34)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Promise.all (index 0) {
  code: 'ERR_IMPORT_ASSERTION_TYPE_MISSING'
}
error Command failed.
Exit code: 1
Command: /usr/local/bin/create-dulo-app
Arguments: 
Directory: /private/tmp/test
Output:

info Visit https://yarnpkg.com/en/docs/cli/create for documentation about this command.
Ricardos-MacBook-Pro-3:test ricardovandenbroek$ 
```

Some possibly useful background information:

* MacBook Pro (14-inch, 2021) with Apple M1 Pro

```
Ricardos-MacBook-Pro-3:test ricardovandenbroek$ node --version
v16.15.0
Ricardos-MacBook-Pro-3:test ricardovandenbroek$ yarn --version
1.22.10
Ricardos-MacBook-Pro-3:test ricardovandenbroek$ npm --version
8.5.5
```

Also, the same happens if I choose vanilla and no TypeScript.

**Comment by jaredtbrown** [comment]
Hi @ricardovdbroek!

Thanks for submitting our very first issue!

Looks like node released import assertions in `17.1.0` and then backported them in `16.14.0` to be required, which is why you are experiencing this issue. [See 16.14.0 release notes here.](https://nodejs.org/en/blog/release/v16.14.0/)

This appears to cause an issue during template generation when importing the templates `package.json` file. Feel free to make a PR to fix if you have the time or I will when I have the time.

**Comment by jaredtbrown** [comment]
@ricardovdbroek a fix has been published in `0.6.2`.
